driver/vhci/gencat.bat

```diff
@@ -6,7 +6,7 @@ copy ..\usbip_vhci.sys
 copy ..\usbip_vhci.cer
 copy ..\usbip_vhci.inf
 inf2cat /driver:.\ /os:%2 /uselocaltime
-signtool sign usbip_vhci.cat
+signtool sign /f %3 /p usbip usbip_vhci.cat

```

Comment:
I store password directly to bat file. If needed i can move it as a script parameter.

Reply:
thats ok by me also, as it uses the file from repository, so thats good for potential ci/cd